### PR TITLE
[Add] Add italian translation, change field class to edit-only for better-zip field

### PR DESCRIPTION
**Pull request by FrancoTampieri**
Hi following this discussion https://github.com/OCA/partner-contact/issues/33 I made a PR.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Hi, Franco, thanks for the changes, but propagate your name as contributor in all files is excessive, having in account that you have only add a class tags in a XML file. This is also an old technique that must be deprecated as soon as possible, because it's very unfair about the proportionality each one contribute. GitHub now tell us what everyone has done.

**Comment by FrancoTampieri** [comment]
Hi Pedro,

no problem, I had seen the lines with the contributors name and I have added myself, but did not know there were any particular policy. I remove my name and made the PR.
I've added IT translation on the PR

Regards

**Comment by scigghia** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1: 

Thanks for the changes.

File: base_location_geonames_import/i18n/it.po [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#: field:better.zip.geonames.import,country_id:0
+msgid "Country"
+msgstr "Paese"
+
+#. module: base_location_geonames_import
+#: code:addons/base_location_geonames_import/wizard/geonames_import.py:66
+#: code:addons/base_location_geonames_import/wizard/geonames_import.py:90
+#, python-format
+msgid "Error:"
+msgstr "Errore :"
```
Comment (tafaRU) [inline base_location_geonames_import/i18n/it.po]: Please, remove whitespace before ':'. Thank you.

Comment (FrancoTampieri) [inline base_location_geonames_import/i18n/it.po]: Hi Tafaru I've fixed the issue in ti.po, can you try it and give me feedback?

Regards

**Comment by tafaRU** [comment]
:+1:

**Comment by OpenCode** [comment]
:+1:

**Comment by tafaRU** [comment]
~~I think Travis build failed due to https://github.com/OCA/partner-contact/pull/31~~
Travis fails after running test_pylint.

**Comment by FrancoTampieri** [comment]
@tafaRU So we have to solve that PR to merge this?

**Comment by FrancoTampieri** [comment]
The problem in thavis is a pylint failure on module passport and base_parter_merge

`************* Module partner-contact.passport.tests.test_passport
passport/tests/test_passport.py:36: [W0102(dangerous-default-value), Base_Test_passport.setUp] Dangerous default value {} as argument`

`************* Module partner-contact.base_partner_merge.base_partner_merge
base_partner_merge/base_partner_merge.py:29: [W0102(dangerous-default-value), html_entity_decode_char] Dangerous default value htmlentitydefs.entitydefs ({}) as argument`

Can I help to fix this issue?

**Comment by tafaRU** [comment]
@drdran you had received more than 3 approves so I think your PR can be merged ;-)
My comment is just meant to point out the reason the Travis build was failing.

**Comment by FrancoTampieri** [comment]
@tafaRU thanks for clearing up the problem:) anyway is if you need a hand to solve the problem with travis renew my call for help

**Comment by tafaRU** [comment]
Thank you @drdran
I'm in favour to merge the PR as is, does anybody have any objections?

**Comment by pedrobaeza** [comment]
OK, we proceed

**Pull request merged**